Description: Fix FindFreetype after headers location breakage.
 Make sure to find freetype headers in their new location (without an
 extra freetype/ directory).
Author: Cyril Brulebois <kibi@debian.org>
Bug-Debian: http://bugs.debian.org/731089

--- a/Modules/FindFreetype.cmake
+++ b/Modules/FindFreetype.cmake
@@ -56,7 +56,7 @@ find_path(FREETYPE_INCLUDE_DIR_ft2build
   PATH_SUFFIXES include/freetype2 include
 )
 
-find_path(FREETYPE_INCLUDE_DIR_freetype2 freetype/config/ftheader.h
+find_path(FREETYPE_INCLUDE_DIR_freetype2 config/ftheader.h
   HINTS
     ENV FREETYPE_DIR
   PATHS
